pocs/scheduler/dispatch.py

```diff
@@ -98,8 +98,9 @@ def get_observation(self, time=None, show_all=False, reread_fields_file=False):
             if self.current_observation is not None:
                 # Favor the current observation if still available
                 end_of_next_set = time + self.current_observation.set_duration
-                if end_of_next_set < common_properties['end_of_night'] and \
-                        self.observation_available(self.current_observation, end_of_next_set):
+                if (end_of_next_set < common_properties['end_of_night'] and
```

Comment:
I'm struggling to decipher this expression. Please add more detail to the preceding comment.